module-services/service-bluetooth/ServiceBluetooth.cpp

```diff
@@ -342,11 +340,14 @@ auto ServiceBluetooth::handle(message::bluetooth::DisconnectResult *msg) -> std:
         bus.sendMulticast(
             std::make_shared<sys::bluetooth::BluetoothModeChanged>(sys::bluetooth::BluetoothMode::Enabled),
             sys::BusChannel::BluetoothModeChanges);
+
+        if (bluetoothDevicesModel->getDevices().size() == 0) {
+            startTimeoutTimer();
+        }
```

Comment:
AFAIK list of devices is not the same as connected devices - IMO this if statement is unnecessary. While handling disconnect message there should be no BT devices connected

Reply:
Fixed.